src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in

```diff
@@ -178,6 +179,10 @@
       "kind": "Sdk",
       "version": "${PackageVersion}"
     },
+    "Microsoft.NET.Sdk.WebAssembly.Pack": {
```

Comment:
What is the consequence of this change? Will it still work in blazor without workload?